demos/JWST/S5_fit_par_template.epf

```diff
@@ -70,6 +70,25 @@ u2          0.84            'free'          0           1           U
 # osc_t0          0.3163  'fixed'         0            1              U
 # osc_t1          0.3163  'free'          0.3          0.4            U
 
+# ------------------
+# ** Star Spot parameters **
+# each spot needs rad, lat, lon. see docs for details!
+# for fleck: can only have one spot contrast (spotcon0), spotnpts is the number of temporal points to evaluate at, spotrad is relative to star
+# for starry: assign one spotcon per spot (spotcon0, spotcon1, etc), spotres is the ydeg of the star map, spotrad is in degrees
+# NOTE: fleck contrast = 1 - starry contrast
+# ------------------
+# spotstari    90       'fixed'         90        1       N
+# spotrot      10       'fixed'         10        1       N
+# spotnpts     100      'independent
+# spotcon0     0.9      'free'          0         1       U
+# spotrad0     0.2      'free'          0         360     U
+# spotlat0     0        'free'          -90       90      U
+# spotlon0     0        'free'          -180      180     U
+# spotcon1     0.1      'free'          0         1       U
```

Comment:
Probably worth adding a comment at the end of this line to remind users that you should only add a `spotcon1` parameter if you were using starry to model two spots since fleck only permits one spot contrast for all spots.

Reply:
done